holium/realm
# Wallet: Transaction Support

Wraps up the Wallet work @leowini and I have been doing the past few weeks.

To test this out:
1. Install the `%wallet` agent using the copy desk script
2. Get your hands on a mnemonic and it's first address (an easy way to generate these is with ganache, but make sure you write it down: by default ganache generates a different address each time it boots up)
3. Take that address and plug it in over [here](https://faucet.egorfine.com/) to load it up with Eth on the ropsen testnet we're using.
4. When opening the wallet app after logging in, select the "import wallet" method on the create screen using your mnemonic from earlier.

At this point you should be good to go. If you boot up a second ship an initialize it's wallet, you'll be able to send funds back and forth.
